File: Baekjoon/Swift/Sorting/Q11652.swift
//
//  Q11652.swift
//  BOJ_PS
//
//  Created by 김태훈 on 2021/06/07.
//  카드 / 정렬

var cards = [Int: Int]()
for _ in 0 ..< Int(readLine()!)! {
    let n = Int(readLine()!)!
    if cards.keys.contains(n) {
        cards[n]! += 1
    } else {
        cards[n] = 1
    }
}

var res = cards.sorted { $0.value == $1.value ? $0.key < $1.key : $0.value > $1.value }
print(res.first!.key)

File: Baekjoon/Python/BOJ_PS/Sorting/Q11652.py
# 카드 / 정렬
cards = {}
for _ in range(int(input())):
    n = int(input())
    
    if n in cards: 
        cards[n] += 1
    else:
        cards[n] = 1

res = sorted(cards.items(), key= lambda x: (-x[1], x[0]))
print(res[0][0])